### ISSUE TITLE
Value List Item nonInteractive prop - Ensure all active elements receive keyboard focus or can be activated with the keyboard - (2036570434)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5623]
### Module: 
 39 Value List and Value List Item 

### Violation: 
 Ensure all active elements receive keyboard focus or can be activated with the keyboard 

![image](https://user-images.githubusercontent.com/5023024/202569006-23b0f52b-b0f5-4cfe-b2a9-cca2ab9caf41.png)

### WCAG Reference: 
- [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) 

### Instance ID: 
 2036570434 

### Severity: 
 8 

### Description: 
 [Issue]
There are actionable elements that do not receive keyboard focus. Examples include:

- River
- Mountain
- Lake
Meadow

[User Impact]
When actionable elements do not receive focus, keyboard-only and screen reader users will not be able to interact with these controls.

[Code Reference] 
 ```
<label aria-label="River" class="label" tabindex="0">
  <div aria-checked="true" class="text-container" role="menuitemcheckbox">
    <span class="title">River</span>
    <span class="description">A large natural stream of water flowing in a channel to the sea, a lake, or another such stream.</span>
  </div>
</label>
 ``` 

### Note: 
 [Suggestion]
Ensure that all actionable controls are in the focus order. It is recommended to use native HTML controls such as ``` <a href="#"> ```, ``` <button> ```, ``` <input type="text"> ``` etc. as they provide keyboard focus by default.

If using custom controls, ensure that tabindex=0 is applied (appropriate event handlers should also be added). When links are missing the href attribute, this can cause them to not be in the focus order.

[Compliant Code Example] 
 ```
<div class="label">
  <div aria-label="River Test 1. Value List" tabindex="0" aria-checked="true" class="text-container" role="menuitemcheckbox">
    <span class="title">River</span>
    <span class="description">A large natural stream of water flowing in a channel to the sea, a lake, or another such stream.</span>
  </div>
</div>
 ```
 

### Media Type: 
 Keyboard Accessibility 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This looks to be a result of the `nonInteractive` prop, which may be out of scope with the updated `list` component. 

Leaving open for confirmation, but likely can be closed once confirmed.

**Comment by driskull** [comment]
This prob was deprecated so this issue is no longer relevant.
